### PR TITLE
added interactive console functionality

### PR DESCRIPTION
**Pull request by RushikeshPatange**
1. Separate Flashing and Console mode with their respective baudrates
2. Interactive Console implementation using SerialTerminal approach
3. Error events handling during flashing device through both default(QuickStart) and manual(DIY) mode
4. IssueHandled: Id append at end of URL in AboutUs page relative href's
5. UI Fix : Fixed Footer position
6. ErrorHandle : Pressing of erase button during flashing
7. BugHandle : ProgramButton disabled until device get connected

### CODE REVIEW

**Comment by igrr** [comment]
Thanks for the PR @RushikeshPatange! To make it easier to preview the new version in the branch, could you please enable Github Pages in your fork and push your changes also to the `main` branch of your repo? Once you do this, we will be able to see the new version in https://rushikeshpatange.github.io/esp-launchpad.

File: package-lock.json [hunk 1]
```diff
@@ -13,35 +13,46 @@
...
-			"version": "2.1.0",
-			"resolved": "https://registry.npmjs.org/pako/-/pako-2.1.0.tgz",
-			"integrity": "sha512-w+eufiZ1WuJYgPXbV/PO3NCMEc3xqylkKHzp8bxp1uW4qaSNQUkwmLLEc3kKsfz8lpV1F8Ht3U1Cm+9Srog2ug=="
+		"node_modules/crypto-js": {
+			"version": "4.1.1",
+			"resolved": "https://registry.npmjs.org/crypto-js/-/crypto-js-4.1.1.tgz",
+			"integrity": "sha512-o2JlM7ydqd3Qk9CA0L4NL6mTzU2sdx96a+oOfPu8Mkl/PK51vSyoi8/rQ8NknZtk44vq15lmhAj9CIAGwgeWKw=="
+		},
+		"node_modules/esptool-js": {
+			"resolved": "../esptool-js",
```
Comment (igrr) [inline package-lock.json:48]: Seems like this might be a local change?

**Comment by igrr** [comment]
@RushikeshPatange Note that in order to have the preview in Github Pages you don't need to check in `node_modules` into the branch. In your fork configuration, enable publishing to Github Pages from Github Actions:
Screenshot 2023-02-17 at 9 14 46

Then the CI workflow in https://github.com/espressif/esp-launchpad/blob/main/.github/workflows/pages.yml will then run, install modules from npm and will finally publish the site to Github Pages.

File: package-lock.json (hunk 1 above)
Comment (RushikeshPatange) [reply package-lock.json]: Thanks, I have fixed it now.

**Comment by RushikeshPatange** [comment]
The PR is ready now. 
@igrr I have enabled publishing to Github Pages from Github Actions.
PTAL: https://rushikeshpatange.github.io/esp-launchpad

**Pull request closed without merging**